lib/autokey/gtkui/configwindow.py

```diff
@@ -428,179 +428,6 @@ def set_dirty(self):
-
-        # Check settings
-        errors += self.settingsWidget.validate()
-
-        if errors:
-            msg = PROBLEM_MSG_SECONDARY % '\n'.join(errors)
-            dlg = Gtk.MessageDialog(
-                self.parentWindow.ui,
-                Gtk.DialogFlags.MODAL | Gtk.DialogFlags.DESTROY_WITH_PARENT,
-                Gtk.MessageType.ERROR,
-                Gtk.ButtonsType.OK,
-                PROBLEM_MSG_PRIMARY
-            )
-            dlg.format_secondary_text(msg)
-            dlg.run()
-            dlg.destroy()
-
-        return len(errors) == 0
-
-    def record_keystrokes(self, isActive):
-        if isActive:
-            self.recorder = autokey.iomediator.keygrabber.Recorder(self)
-            dlg = dialogs.RecordDialog(self.ui, self.on_rec_response)
-            dlg.run()
-        else:
-            self.recorder.stop()
-
-    def on_rec_response(self, response, recKb, recMouse, delay):
-        if response == Gtk.ResponseType.OK:
-            self.recorder.set_record_keyboard(recKb)
-            self.recorder.set_record_mouse(recMouse)
-            self.recorder.start(delay)
-        elif response == Gtk.ResponseType.CANCEL:
-            self.parentWindow.record_stopped()
-
-    def cancel_record(self):
-        self.recorder.stop()
-
-    def start_record(self):
-        self.buffer.insert(self.buffer.get_end_iter(), "\n")
-
-    def start_key_sequence(self):
-        self.buffer.insert(self.buffer.get_end_iter(), "keyboard.send_keys(\"")
-
-    def end_key_sequence(self):
-        self.buffer.insert(self.buffer.get_end_iter(), "\")\n")
-
-    def append_key(self, key):
-        #line, pos = self.buffer.getCursorPosition()
-        self.buffer.insert(self.buffer.get_end_iter(), key)
-        #self.scriptCodeEditor.setCursorPosition(line, pos + len(key))
-
-    def append_hotkey(self, key, modifiers):
-        #line, pos = self.scriptCodeEditor.getCursorPosition()
-        keyString = self.currentItem.get_hotkey_string(key, modifiers)
-        self.buffer.insert(self.buffer.get_end_iter(), keyString)
-        #self.scriptCodeEditor.setCursorPosition(line, pos + len(keyString))
-
-    def append_mouseclick(self, xCoord, yCoord, button, windowTitle):
-        self.buffer.insert(self.buffer.get_end_iter(), "mouse.click_relative(%d, %d, %d) # %s\n" % (xCoord, yCoord, int(button), windowTitle))
-
-    def undo(self):
-        self.buffer.undo()
-        self.parentWindow.set_undo_available(self.buffer.can_undo())
-        self.parentWindow.set_redo_available(self.buffer.can_redo())
-
-    def redo(self):
-        self.buffer.redo()
-        self.parentWindow.set_undo_available(self.buffer.can_undo())
-        self.parentWindow.set_redo_available(self.buffer.can_redo())
-
-    def on_modified(self, widget, data=None):
-        self.set_dirty()
-        self.parentWindow.set_undo_available(self.buffer.can_undo())
-        self.parentWindow.set_redo_available(self.buffer.can_redo())
```

Comment:
Did you mean to remove all of this code?

Reply:
Oh dang, thank you. This didn't even look changed to me, and definitely shouldn't have been removed